Assets/MRTK/SDK/Features/UX/Scripts/Utilities/PrefabSpawner.cs

```diff
@@ -0,0 +1,239 @@
+﻿using Microsoft.MixedReality.Toolkit.Input;
```

Comment:
Since this script largely contains code from the other licensed file, we should likely have the license here as well.

```suggestion
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

﻿using Microsoft.MixedReality.Toolkit.Input;
```